src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -15,13 +15,14 @@ import keycodes from '../../keycodes'
 import { EthereumAddressType } from '../../prop-types'
 
 const LocalIdentityModal = React.memo(
-  ({ opened, address, label, onCancel, onSave }) => {
+  ({ opened, address, label, onCancel, onDelete, onSave }) => {
```

Comment:
There's an ESLint annotation here, but I also think we could just combine `LocalIdentityModal` with `LocalModal`.

There's a bit of history behind this component, but suffice to say, its implementation was more complex before and it made more sense to split up the view / controller back then.